react_rails_getting_started.org

```diff
@@ -33,37 +35,19 @@ gem 'opal-rails' # already added w/the --javascript=opal option
 gem 'therubyracer', platforms: :ruby # Required for prerendering
 #+END_SRC
 
-Add these and run =bundle install=
+Then run =bundle install=
 
-* Convert application.js to application.rb
+** Configure application.rb
 
-To support ordered requires, starting with Opal 0.8.*, if one does NOT
-use application.rb, one needs to manually load each opal file in the
-application.js, as below.
-
-#+BEGIN_SRC javascript
-// application.js
-//= require opal
-//= require greeter
-//= require_self
-Opal.load('an_opal_file');
-Opal.load('another_opal_file');
-// etc.
-#+END_SRC
-
-Prior to 0.8.* you didn't not need to user =application.rb= in lieu of
-=application.js=. As such, it is much more convenient to convert the
-=application.js= to =application.rb=.  =application.rb= will
-automatically load the the files that are =require=d without an extra
-=Opal.load()= step.  You can do it quickly with following steps (these
-steps are for *nix operating systems (i.e. linux, BSD/OSX), where the
-^D is ctrl-D, the EOF keystroke which will finish the cat >
-app/assets/javascripts/application.js.rb command - kindly adjust for
-other operating systems).
```

Comment:
I don't think the "how" discussion adds anything to the blog post; recommend leaving it out.  If you're really attached to it, put it in an addendum or footnote.